lib/travis/build/script/objective_c.rb

```diff
@@ -59,9 +59,19 @@ def install
           sh.if podfile? do
             sh.if "! ([[ -f #{pod_dir}/Podfile.lock && -f #{pod_dir}/Pods/Manifest.lock ]] && cmp --silent #{pod_dir}/Podfile.lock #{pod_dir}/Pods/Manifest.lock)", raw: true do
               sh.fold('install.cocoapods') do
-                sh.echo "Installing Pods with 'pod install'", ansi: :yellow
+
                 sh.cmd "pushd #{pod_dir}"
-                sh.cmd 'pod install', retry: true
+
+                sh.if gemfile? do
+                  sh.echo "Installing Pods with 'bundle exec pod install'", ansi: :yellow
+                  sh.cmd 'bundle exec pod install', retry: true
+                end
+
+                sh.elif gemfile? do
```

Comment:
This `elif` condition seems identical to the immediately preceding `if` condition. I don't think it works.